core/oracles/oracle_spec.go

```diff
@@ -40,6 +40,8 @@ var (
 	ErrMissingPropertyName = errors.New("a property name is required")
 	// ErrInvalidPropertyKey is returned if validation finds a reserved Vega property key.
 	ErrInvalidPropertyKey = errors.New("property key is reserved")
+	// ErrMultipleUniqueNamesInFilterList is returned if filters with same key names exists inside a single list.
+	ErrMultipleUniqueNamesInFilterList = errors.New("multiple unique keys with same name found in filter list")
```

Comment:
maybe remove the *unique*

Reply:
Made it like this everywhere:
`
ErrMultipleSameKeyNamesInFilterList = errors.New("multiple keys with same name found in filter list")

`